backend/src/controllers/dummyapi.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs')
+
+async function dummyApi(req, res) {
+    try {
+        let data = req.body
+        let requiredData = Object.keys(data)
+            .map((key, value) => { return [key, data[key]].join('='); })
+            .join('\n');
+        fs.writeFile('data.txt', `Date: ${new Date().toLocaleString()} \n${requiredData}\n\n`, { flag: 'a+' }, err => {
+
+            if (err) {
+               return res.send("err", err)
+            }
+            // console.log(new Date());
+        })
+        res.status(200).json({ statusMessage: "Success", message: "Data Saved Successfully!!!", data: data })
```

Comment:
send like this res.success("Data saved successfully!!!", {data}) and why we are sending req.body data again to frontend

Reply:
sir i will change this afterwards